app/services/form_service.py

```diff
@@ -22,29 +22,35 @@ def create_form(self, ong_id:str, animal_id:str, form: FormModel, request_id:str
             with self.db.session.start_transaction():
                 response = self.ong_service.get_ong_animals(ong_id)
                 if response.status != http.HTTPStatus.OK or not response.data: #baseado no reponse.data
-                    #TODO; logging nao tem animal na ong
+                    self.logger.error(f"id = {request_id} There is no animal in Ong")
                     return ResponseDTO(None, "Ong has no animals", http.HTTPStatus.BAD_REQUEST)
                 animals = response.data
                 animal_exist = False
                 for animal in animals:
                     if str(animal.get("id")) == animal_id:
+                        self.logger.info(f"id = {request_id} Found Animal")
                         animal_exist = True
                         break
                 if not animal_exist:
+                    self.logger.info(f"id = {request_id} Animal not Found")
                     return ResponseDTO(None, "Animal doesn't belongs to ONG. Aborting.", http.HTTPStatus.BAD_REQUEST)
                 form.animal_id = animal_id
+                self.logger.info(f"id = {request_id} Inserting form in Collection")
                 result = self.form_collection.insert_one(form.model_dump())
                 if result:
+                    self.logger.info(f"id = {request_id} Insertion Sucessfull, inserting in animal")
                     response = self.animal_service.insert_form(animal_id, result.inserted_id)
                     # TODO: tratar rollback
                     if response.status != http.HTTPStatus.OK:
                         return response
                     #new_form = self.form_collection.find_one(result.inserted_id)
-                    return ResponseDTO({"id": result.inserted_id},"Form created successfully", http.HTTPStatus.CREATED)
+                    self.logger.info(f"id= {request_id} Form Create Sucessfully")
+                    return ResponseDTO({"id": str(result.inserted_id)},"Form created successfully", http.HTTPStatus.CREATED)
                 else:
                     return ResponseDTO(None, "Couldn't Create Form. Aborting.", http.HTTPStatus.BAD_REQUEST)
         except Exception as e:
             # TODO:Utilizar a biblioteca logging para criar uma documentação clara do que esta rolando na api. Nota: Isso facilita o debug e rastreabilidade tmb
+            self.logger.error(f"id= {request_id} Error creating form {e}")
             print(f"Error creating Forms: {e}")
             return ResponseDTO(None, "Error Creating Form: " + str(e), http.HTTPStatus.BAD_REQUEST)
```

Comment:
O Usuário (Cliente/Frontend) nao dseve saber detalhes de erros do backend. os logs sao para filtragem em produção, dps te mostro o deploy no render.

```suggestion
            return ResponseDTO(None, "Error Creating Form", http.HTTPStatus.BAD_REQUEST)
```